SPECS/ruby/ruby.spec

```diff
@@ -1,21 +1,88 @@
 # The RubyGems library has to stay out of Ruby directory tree, since the
 # RubyGems should be share by all Ruby implementations.
 %global rubygems_dir  %{_datadir}/rubygems
-%global ruby_libdir   %{_datadir}/%{name}
 
-%global gem_dir %{_libdir}/ruby/gems
-%global rubygems_molinillo_version    0.5.7
+%global gem_dir %{_datadir}/ruby/gems
 %global rubygems_version              3.1.6
 
-%global rubygems_minitest_version     5.15.0
-%global rubygems_power_assert_version 2.0.1
-%global rubygems_rake_version         13.0.6
-%global rubygems_test_unit_version    3.5.3
+# Add version for default gems
+%global abbrev_version     0.1.0
```

Comment:
Do you by chance have a script or any other automation we can run to update these versions whenever we update `ruby`? If not, at the very least, please add a comment here mentioning where to retrieve these versions from and one next to the `Version` tag with a warning to always make sure these values are correct.

Reply:
Updated comment in spec file. I don't have a script, but can write one later and add it to our wiki